### DISCUSSION TITLE
.kitty-ssh-kitten-untar-jA6qnetXw5mY/bootstrap-utils.sh: No such file or directory

### DISCUSSION THREAD

**Discussion by zhenhua-wang** [kovidgoyal/kitty#7548] [Q&A] [answered]
I got this error when `kitty +kitten ssh` into my remote server. I can't figure out any possible cause. I really appreciate any help on this. Thank you.

My kitty version is 0.35.1-1 on arch.

**Comment by kovidgoyal** [comment, marked as answer]
It usually is caused by tar on the remote server failing. You can try
using python as the remote interpreter instead of sh that might work or
at least give you a more useful failure message,

https://sw.kovidgoyal.net/kitty/kittens/ssh/#opt-kitten-ssh.interpreter

**Reply by zhenhua-wang** [reply]
Thanks for your information. It does give a useful message. I was having some permission issues on my server. Now everything works well.
